cli/cycpp.py

```diff
@@ -746,6 +748,7 @@ def includeloc(self, statement=None):
         'std::list': ('T',),
         'std::pair': ('T1', 'T2'),
         'std::map': ('Key', 'T'),
+        'std::tuple': ('T',),
```

Comment:
So this is incorrect because tuples have an arbitrary number of template args, not 1.  We need to have some way of denoting this. Maybe we need a special list type that returns 'T{n}' for the nth index.